ui/app/components/app/transaction-list-item/transaction-list-item.component.js

```diff
@@ -2,6 +2,7 @@ import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import classnames from 'classnames'
 import Identicon from '../../ui/identicon'
```

Comment:
remove the changes in this file I think

Reply:
I had meant not to add this file, but accidentally did in my last lint nit commit, should be good now.